core/forking/gasSchedule.go

```diff
@@ -44,6 +44,7 @@ func NewGasScheduleNotifier(args ArgsNewGasScheduleNotifier) (*gasScheduleNotifi
 		handlers:          make([]core.GasScheduleSubscribeHandler, 0),
 		configDir:         args.ConfigDir,
 	}
+	log.Debug("gasSchedule: enable epoch for gas schedule directories paths epoch", g.gasScheduleConfig.GasScheduleByEpochs)
```

Comment:
this is a wrongly formatted log line. We should always have an odd number of parameters:
<log_string><text_1><value_1><text_2><value_2>....<text_n><value_n>

Second: what is the point of printing the whole loaded structure?

Reply:
done